src/bzz.js

```diff
@@ -2,29 +2,28 @@
 
 import debug from 'debug'
 import { BZZ } from 'erebos'
-import { buffer, send } from 'micro'
-import { get, post } from 'microrouter'
+import { buffer } from 'micro'
+import type { express$Application } from 'express'
```

Comment:
Does this work? Flow doesn't complain about it?
From my understanding we whould add express to the `flow-typed` folder by running `flow-typed install`, and then then `express$Application` should be available to Flow with no need to import it.

Reply:
flow didn't complain but I wasn't aware of adding it to flow-typed so will update